gocd-filebased-authentication-plugin/build.gradle

```diff
@@ -15,11 +15,12 @@
  */
 
 dependencies {
-  compileOnly group: 'cd.go.plugin', name: 'go-plugin-api', version: '17.2.0'
+  compileOnly group: 'cd.go.plugin', name: 'go-plugin-api', version: '18.1.0'
   compile group: 'com.google.code.gson', name: 'gson', version: '2.8.0'
   compile project(':bcrypt')
+  compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.0'
```

Comment:
Why was this required?

Reply:
@ketan To compile with JDK 9 and above. They removed some stuff in it:

http://mail.openjdk.java.net/pipermail/jdk9-dev/2016-May/004309.html
https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j

We've done this before, in another plugin too: https://github.com/gocd-contrib/go-nuget-poller-plugin-2.0/commit/9b8bd794af54f202efd7aa9defc9a6583637dccd#diff-c197962302397baf3a4cc36463dce5eaL46

Reply:
Here's an example of the usage in this code: https://github.com/gocd/gocd-filebased-authentication-plugin/blob/672cb0f31ff6fda25cf9c3a1babb07aab45d52fd/gocdpasswd/src/main/java/cd/go/authentication/crypt/hash/PBKDF2Provider.java#L23